modules/ossm-cr-example.adoc

```diff
@@ -3,26 +3,141 @@
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
 
-The {ProductShortName} operator extends the API by adding the `ServiceMeshControlPlane` resource type, which enables you to create `ServiceMeshControlPlane` objects within projects. By creating a `ServiceMeshControlPlane` object, you instruct the Operator to install a {ProductShortName} control plane into the project, configured with the parameters you set in the `ServiceMeshControlPlane` object.
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for the `ServiceMeshControlPlane` version 2.0 is `maistra.io/v2`.
+|The value for `ServiceMeshControlPlane` version 2.0 is `maistra.io/v2`.
 
-This example `ServiceMeshControlPlane` definition contains all of the supported parameters and deploys {ProductName} {ProductVersion} images based on Red Hat Enterprise Linux (RHEL).
+|`kind`
+|Kind is a string value representing the REST resource this object represents.
+|`ServiceMeshControlPlane` is the only valid value for a ServiceMeshControlPlane.
 
-[IMPORTANT]
-====
-The 3scale Istio Adapter is deployed and configured in the custom resource file. It also requires an active 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/infrastructure/onpremises-installation[On-Premises]).
-====
+|`metadata`
+|Metadata about this `ServiceMeshControlPlane` instance You can provide a name for your control plane installation to keep track of your work, for example, `basic`.
+|string
 
-.Example `ServiceMeshControlPlane` resource
+|`spec`
+|The specification of the desired state of this `ServiceMeshControlPlane`. This includes the configuration options for all components that comprise the control plane.
+|For more information, see Table 2.
+
+|`status`
+|The current status of this `ServiceMeshControlPlane` and the components that comprise the control plane.
+|For more information, see Table 3.
+|===
 
+The following table lists the specifications for the `ServiceMeshControlPlane` resource. Changing these parameters configure {ProductName} components.
+
+.`ServiceMeshControlPlane` resource spec
+|===
+|Name |Description |Configurable parameters
+
+|`addons`
+| The `addons` parameter configures additional features beyond core control plane components, such as visualization, or metric storage.
+|`3scale`, `grafana`, `jaeger`, `kiali`, and `prometheus`.
+
+|`cluster`
+|The `cluster` parameter sets the general configuration of the cluster (cluster name, network name, multi-cluster, mesh expansion, etc.)
+|`meshExpansion`, `multiCluster`, `name`, and `network`
+
+|`gateways`
+| You use the `gateways` parameter to configure ingress and egress gateways for the mesh.
```

Comment:
Parallelism. Some descriptions begin by repeating the name of the parameter. Others begin with "you use..."
I recommend starting with the third-person form of the verb:
* Configures additional features...
* Sets the general configuration...
* Configures the ingress and egress gateways...